docs/features/telemetry-enrichment.md

```diff
@@ -24,6 +24,16 @@ Value: `1.0.0-preview`
 
 **Usage**
 
+```csharp
+ILogger logger = new LoggerConfiguration()
+    .Enrich.WithVersion()
+    .CreateLogger();
+
+logger.Information("This event will be enriched with the runtime assembly product version");
+```
+
+Or using the enricher directly:
+
 ```csharp
 ILogger logger = new LoggerConfiguration()
     .Enrich.With<VersionEnricher>()
```

Comment:
Do we need to keep this approach documented? I'm more leaning towards removing it to make the docs more condensed but happy to hear your opinion on this

Reply:
Yean, you might be right. I'll remove this.